### ISSUE TITLE
Request: Allow track layers to place suspended tracks on top of air blocks

### ISSUE THREAD

**Issue by MalkContent** [Railcraft/Railcraft#549]
Be aware that this would allow automatically building bridges when the track layer is accompanied by an undercutter (which already can replace air blocks).

**Comment by liach** [comment]
Nice idea! But I afraid the track layer cart would go down before the suspended track has been laid. Force track emitters are better, functions almost the same.

**Comment by MalkContent** [comment]
I don't think it would go down. I think the issue is that it only tries to place rails when there would be a full block beneath it.
Force track emitters do not serve the same purpose, I'm afraid.

**Comment by liach** [comment]
Pinging @Yopu for further explaination...

**Comment by CovertJaguar** [comment]
You'd still eventually end up with your Track Layer at the bottom of a hole somewhere. Suspended Tracks can only extend two blocks out from a support.

**Comment by MalkContent** [comment]
when you have an undercutter laying down supports behind it, it's possible.

**Comment by Yopu** [comment]
Seems like a really odd edge case. Let me see how hard it would be to implement.

**Comment by Yopu** [comment]
From initial tests it might be doable. However, @MalkContent you do realize that it wouldn't be able to traverse vertically since the suspend track can't be sloped.

**Comment by MalkContent** [comment]
That's okay. It's supposed to be free floating anyways, it you run it into a hill, you did something wrong. Plus you can only ever put 64 tracks into the layer since there's no room for a chest cart.

**Comment by Yopu** [comment]
This looks to be more work than I have time for right now.

**Comment by MalkContent** [comment]
No bother. I thought it was only as much work as allowing the track 
layer to place suspended tracks on air. I think I can live and die 
without this request being fulfilled :)

Am 19.03.2015 um 14:44 schrieb Yopu:
>
> This looks to be more work than I have time for right now.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/CovertJaguar/Railcraft/issues/549#issuecomment-83586934>.
>



---
Diese E-Mail wurde von Avast Antivirus-Software auf Viren geprüft.
http://www.avast.com

**Comment by DarianLStephens** [comment]
Aw, dang. This would have been perfect for an automatic sky bridge rail. I'd place suspended track, put a block under it, and then re-lay normal tracks in its place. It's a shame this doesn't seem to be possible, that would have been a great way to use these things.
Is there any way to automatically lay a skybridge, without using other mods?

**Comment by liach** [comment]
@darianlstephens the force track emitter will be back.

**Comment by CovertJaguar** [comment]
The skybridge thing is kind of cool. I'll poke it eventually. Probably once @liach is done with the relayer rework.
